pkgs/tools/misc/rpi-imager/default.nix

```diff
@@ -0,0 +1,45 @@
+{ mkDerivation,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  curl,
+  libarchive,
+  util-linux,
+  qt5
+}:
+
+mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
I did this at first, but [chapter 15.20 of the Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-language-qt) says otherwise:

> Use mkDerivation instead of stdenv.mkDerivation. mkDerivation is a wrapper around stdenv.mkDerivation which applies some Qt-specific settings. This deriver accepts the same arguments as stdenv.mkDerivation; refer to Chapter 6, The Standard Environment for details.

Maybe we need some more clarification there.

Reply:
Good catch! You are totally right.